### PR TITLE
Added CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by checor**
This is my attempt to create a contributing file for this project.

Since this is a getting started, I made the file more general about contributing to projects. I hope this is what's intended.

Also, please note that English is not my 1st language. If there are errors, feel free to tell me.

Greetings!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#Contributing to this project
+
+Thanks for taking the time to contribute!
+
+The following is a set of guidelines to for contributing to a project. Feel free to propose change to this document in a pull request.
+
+## Issues
+
+Issues are a great way to keep track of tasks, enhancements, and bugs for projects. If you find a bug, or have a suggestion for enhacement, feel free to open a issue. You can open one using the green button "New issue", located in the "Issues" tab or the project.
```
Comment (prateekiiest) [inline CONTRIBUTING.md]: enhancement : spelling error

Comment (prateekiiest) [reply CONTRIBUTING.md]: open an issue

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+### What a a great issue has
```
Comment (prateekiiest) [inline CONTRIBUTING.md]: remove one 'a' here

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+So what do you need to fill in a issue:
+ * Title: The title should describe the bug or enhacement briefly.
```
Comment (prateekiiest) [inline CONTRIBUTING.md]: enhancement : spelling error

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+ * Labels: Check if the project has labels and add the labels that describes the issue.
+ * Summary: Summarize the bug/enhacement concisely
```
Comment (prateekiiest) [inline CONTRIBUTING.md]: enhancement

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+ * Steps to reproduce (Bugs only): How the developer or others users can reproduce the bug?
+ * Revelant information: Add logs, or screenshots if filling a bug, or maybe a mockup if it's a enhacement.
```
Comment (prateekiiest) [inline CONTRIBUTING.md]: enhancement

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+ * Enviroment info: Tell the version of software and OS you encountered the issue.
```
Comment (prateekiiest) [inline CONTRIBUTING.md]: Environment

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+## Branching strategy
+
+Every project follows a branching strategy, that means, some rules on what each branch name means, and how to name them.
+
+One of the most used branching strategy in Github is [GitHub Flow](https://guides.github.com/introduction/flow/), where you create a branch from the default branch (usually 'master'), put your changes, and the send a PR for discussion. That way, your changes can be reviewed before being accepted.
+
+As a rule of thumb, don't add new changes to 'master', even if the project you are contributing doesn't specify which branching strategy is using.
+
+Happy hacking!
```
Comment (prateekiiest) [inline CONTRIBUTING.md]: Happy Open Sourcing maybe (just a suggestion) 😃

**Review by prateekiiest — CHANGES_REQUESTED** [review]
Well written and detailed.
That's great.
Just a few changes to be made - I have notified.
And its then good to be merged 👍

File: CONTRIBUTING.md (hunk 7 above)
Comment (checor) [reply CONTRIBUTING.md]: Cool! I will add that

**Comment by checor** [comment]
I have corrected the spelling and the double 'a' I put on.

Feel free to take another view!

**Review by prateekiiest — APPROVED** [review]
Perfect!
Thanks @checor :1st_place_medal:

**Pull request merged**